pkg/storegateway/chunk_bytes_pool.go

```diff
@@ -22,14 +21,10 @@ func newChunkBytesPool(minBucketSize, maxBucketSize int, maxChunkPoolBytes uint6
 
 	return &chunkBytesPool{
 		pool: upstream,
-		requestedBytes: promauto.With(reg).NewCounter(prometheus.CounterOpts{
-			Name: "cortex_bucket_store_chunk_pool_requested_bytes_total",
-			Help: "Total bytes requested to chunk bytes pool.",
-		}),
-		returnedBytes: promauto.With(reg).NewCounter(prometheus.CounterOpts{
-			Name: "cortex_bucket_store_chunk_pool_returned_bytes_total",
-			Help: "Total bytes returned by the chunk bytes pool.",
-		}),
+		poolByteStats: promauto.With(reg).NewCounterVec(prometheus.CounterOpts{
+			Name: "cortex_bucket_store_chunk_pool_operation_bytes_total",
+			Help: "Total bytes number of bytes pooled by operation.",
```

Comment:
My suggestion for this would be "Bytes checkout or returned to the chunk pool"

Reply:
humm.. idk.. i think s too verbose